api.rst

```diff
@@ -55,7 +68,25 @@ not           negates another operator, see below
 ============  =============================================
 
 
-To negate any operator, prefix it with :code:`not` like :code:`?a=not.eq.2`.
+To negate any operator, prefix it with :code:`not` like :code:`?a=not.eq.2` or :code:`?not.and=(a.gte.0,a.lte.100)` .
+
+For more complicated filters (such as those involving disjunctions) you will have to create a new view in the database, or use a stored procedure. For instance, here's a view to show "today's stories" including possibly older pinned stories:
```

Comment:
Probably remove the comment "(such as those involving disjunctions)" since we can do that directly now and don't need a stored proc for it.